diff --git a/config/webpackDevServer.config.js b/config/webpackDevServer.config.js
index bf8c98d0d912e610f8c0b78eb30c29ce0c1627ca..1b2acf7ccb77108d89e2d288b7f778477b63cad1 100644
--- a/config/webpackDevServer.config.js
+++ b/config/webpackDevServer.config.js
@@ -43,7 +43,7 @@ module.exports = function (proxy, allowedHost) {
     disableHostCheck:
       !proxy || process.env.DANGEROUSLY_DISABLE_HOST_CHECK === 'true',
     // Enable gzip compression of generated files.
-    compress: true,
+    compress: false,
     // Silence WebpackDevServer's own logs since they're generally not useful.
     // It will still show compile warnings and errors with this setting.
     clientLogLevel: 'none',
